src/components/Layout/Layout.tsx

```diff
@@ -1,5 +1,7 @@
 import { Layout as Container, InputSearch, Sidebar } from '@ebs-integrator/react-ebs-ui';
 import { NavLink } from 'react-router-dom';
+import { routes } from '../../routes';
```

Comment:
Try to use absolute paths, like: `import { routes } from 'routes';`
For this you need add `baseUrl` param in your tsconfig.ts, like:
```javascript
{
    ...,
    "baseUrl": "src",
    ...
}
```